### PR TITLE
 MPP-1791 Replace homepage use cases: new infographics (new strings)

### PR DESCRIPTION
**Pull request by lloan**
Replace homepage use cases: new infographics (new strings)

@Vinnl @maxxcrawford 
Do you have any suggestions or input on the names for these strings or a specific pattern you want to follow?

### CODE REVIEW

File: en/app.ftl [hunk 1]
```diff
@@ -233,6 +233,35 @@ landing-use-cases-gaming = Gaming
...
+landing-use-cases-on-the-go-connect-image-alt = Image of a person sitting on a table looking at their tablet
+landing-use-cases-on-the-go-receipt-image-alt = Image of a person on their phone in front of a store cart
+landing-use-cases-on-the-go-phone-image-alt = Image of a person checking their phone 
+
+landing-use-cases-signups = Signups
+landing-use-cases-signups-hero-heading = Worry-free signups
+landing-use-cases-signups-hero-content1 = Want to start a new subscription, RSVP to an event, or get a bargain promo code without having spam flooding your inbox?
+landing-use-cases-signups-hero-content2 = Before you complete that next signup, use an email mask instead of your real one to protect your info and keep control over your inbox.
+
+landing-use-cases-shopping-image-alt = Image of phone for shopping tab
```
Comment (lloan) [inline en/app.ftl:259]: @Vinnl @maxxcrawford alt strings for images - do we want these to say anything specific? How do you normally go about these?

File: en/app.ftl [hunk 2]
```diff
@@ -233,6 +233,35 @@ landing-use-cases-gaming = Gaming
...
 
+landing-use-cases-shopping-hero-heading = Shopping with email masks
+landing-use-cases-shopping-hero-content1 = Want to buy something online and don’t know or fully trust the shop?
+landing-use-cases-shopping-hero-content2 = Use an email mask whenever you shop online. Get the confirmation sent to your real email and then easily turn the mask off anytime later.
+
+landing-use-cases-on-the-go = On the Go
+landing-use-cases-on-the-go-heading = On the go with { -brand-name-relay }
+landing-use-cases-on-the-go-lead = Instantly make a custom email mask anywhere and everywhere you go! 
+landing-use-cases-on-the-go-connect-heading = Connect on the go
+landing-use-cases-on-the-go-connect-body = Use your email mask when you want to privately sign into your favorite coffee shop or public wifi
```
Comment (flodolo) [inline en/app.ftl:244]:
```suggestion
landing-use-cases-on-the-go-connect-body = Use your email mask when you want to privately sign into your favorite coffee shop or public Wi-Fi 
```

https://mana.mozilla.org/wiki/pages/viewpage.action?spaceKey=FIREFOX&title=Word+List#WordList-Wi-Fi

File: en/app.ftl [hunk 3]
```diff
@@ -233,6 +233,35 @@ landing-use-cases-gaming = Gaming
...
+landing-use-cases-on-the-go-heading = On the go with { -brand-name-relay }
+landing-use-cases-on-the-go-lead = Instantly make a custom email mask anywhere and everywhere you go! 
+landing-use-cases-on-the-go-connect-heading = Connect on the go
+landing-use-cases-on-the-go-connect-body = Use your email mask when you want to privately sign into your favorite coffee shop or public wifi 
+landing-use-cases-on-the-go-receipt-heading = Get email receipts
+landing-use-cases-on-the-go-receipt-body = Share a custom email mask for in-store shopping receipts without sharing your real email 
+landing-use-cases-on-the-go-phone-heading = Use on your phone
+landing-use-cases-on-the-go-phone-body = No matter where you are create a custom email mask in seconds for anything you want to do 
+
+landing-use-cases-on-the-go-connect-image-alt = Image of a person sitting on a table looking at their tablet
```
Comment (flodolo) [inline en/app.ftl:250]: Please check with accessibility folks, but I don't think you need to say "Image …", just "Person sitting…".

File: en/app.ftl [hunk 4]
```diff
@@ -233,6 +233,35 @@ landing-use-cases-gaming = Gaming
...
+landing-use-cases-on-the-go-receipt-image-alt = Image of a person on their phone in front of a store cart
+landing-use-cases-on-the-go-phone-image-alt = Image of a person checking their phone 
+
+landing-use-cases-signups = Signups
+landing-use-cases-signups-hero-heading = Worry-free signups
+landing-use-cases-signups-hero-content1 = Want to start a new subscription, RSVP to an event, or get a bargain promo code without having spam flooding your inbox?
```
Comment (flodolo) [inline en/app.ftl:256]: We had a similar string recently, and `RSVP` was changed to `respond to an invitation`
https://github.com/mozilla/blurts-server/pull/2541

Either that, or we need a localization note.

File: en/app.ftl [hunk 5]
```diff
@@ -233,6 +233,35 @@ landing-use-cases-gaming = Gaming
...
+landing-use-cases-signups-hero-content2 = Before you complete that next signup, use an email mask instead of your real one to protect your info and keep control over your inbox.
+
+landing-use-cases-shopping-image-alt = Image of phone for shopping tab
+landing-use-cases-social-networks-image-alt = Image of thumb up and chat bubbles for social networks tab
```
Comment (flodolo) [inline en/app.ftl:260]: Similarly to the previous comment: I don't think you should add in which tab it's used. The user will already know in which tab they are.

File: en/app.ftl (hunk 3 above)
Comment (Vinnl) [reply en/app.ftl:250]: In fact. in this case I'd even leave the alt empty - no extra information is conveyed in the image (i.e. it's purely decorative), so adding an alt text will only slow people using screen readers down.

See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Img#attr-alt:

> Setting this attribute to an empty string (`alt=""`) indicates that this image is not a key part of the content (it's decoration or a tracking pixel), and that non-visual browsers may omit it from [rendering](https://developer.mozilla.org/en-US/docs/Glossary/Rendering_engine).

File: en/app.ftl (hunk 1 above)
Comment (Vinnl) [reply en/app.ftl:259]: See https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/78#discussion_r862741043

(And yeah, normally I come up with what I hope is a relevant description myself, unless I already have a need to gather feedback from content design. Might be a good idea to bring that up there more often though, to build a habit of coming up with alt strings, if necessary.)

**Comment by Vinnl** [comment]
>  Do you have any suggestions or input on the names for these strings or a specific pattern you want to follow?

That's actually a good question; I've mostly been going with something that looks similar to what's already there, but there might very well be a convention that I'm not aware of yet.

File: en/app.ftl (hunk 1 above)
Comment (lloan) [reply en/app.ftl:259]: removed

File: en/app.ftl (hunk 3 above)
Comment (lloan) [reply en/app.ftl:250]: removed

File: en/app.ftl (hunk 5 above)
Comment (lloan) [reply en/app.ftl:260]: removed

**Review by flodolo — APPROVED** [review]
This looks good to me now.

Are there more content updates planned for this week? If so, we should probably try to wait and merge at the same time, otherwise we risk sending too many notifications to localizers.

File: en/app.ftl (hunk 4 above)
Comment (lloan) [reply en/app.ftl]: We are going to take Monitor's approach and replace `RSVP` - thanks for pointing that out!

**Comment by lloan** [comment]
> anned for this week? If so, we should probably try to wait and merge at the same time, otherwise we risk sending too man

Yes, I have another task or 2 that will require adding some strings - we could merge these at the end of the week if you think that works?

**Comment by flodolo** [comment]
> > anned for this week? If so, we should probably try to wait and merge at the same time, otherwise we risk sending too man
> 
> Yes, I have another task or 2 that will require adding some strings - we could merge these at the end of the week if you think that works?

Yep, that sounds good.

Please add new strings in a separate PR (or PRs), and let me know when we're good to merge.

**Comment by flodolo** [comment]
(I'm going to rebase this against main, just to test that the linter works as expected)

**Comment by Vinnl** [comment]
@lloan Could you also merge https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/80 when you merge this one?

File: en/app.ftl [hunk 6]
```diff
@@ -232,6 +232,25 @@ landing-use-cases-gaming = Gaming
...
 
+landing-use-cases-shopping-hero-heading = Shopping with email masks
+landing-use-cases-shopping-hero-content1 = Want to buy something online and don’t know or fully trust the shop?
+landing-use-cases-shopping-hero-content2 = Use an email mask whenever you shop online. Get the confirmation sent to your real email and then easily turn the mask off anytime later.
+
+landing-use-cases-on-the-go = On the Go
+landing-use-cases-on-the-go-heading = On the go with { -brand-name-relay }
+landing-use-cases-on-the-go-lead = Instantly make a custom email mask anywhere and everywhere you go!
+landing-use-cases-on-the-go-connect-heading = Connect on the go
+landing-use-cases-on-the-go-connect-body = Use your email mask when you want to privately sign into your favorite coffee shop or public Wi-Fi
```
Comment (flodolo) [inline en/app.ftl:243]: Noticed this only when translating: this should have been "sign in to" https://mana.mozilla.org/wiki/pages/viewpage.action?spaceKey=FIREFOX&title=Word+List#WordList-signin

**Pull request merged**